presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -198,3 +198,63 @@ Examples::
     SELECT * FROM (VALUES ('a%c'), ('%cd'), ('cde')) AS t (name)
     WHERE name LIKE '%#%%' ESCAPE '#'
     --returns 'a%c' and  '%cd'
+
+
+
+## Row comparison: IN
+
+The `IN` comparison operator in SQL is used to compare a value with a list of literal values that have been specified.
+The `IN` operator returns true if the value matches any of the literal values in the list.The `IN` operator can be
+used to compare values with the following patterns.It can fetch records according to multiple values specified in
+WHERE clause. A sub-query or list of values must be specified in the parenthesis.
+
+```
+... WHERE column [NOT] IN ('value1','value2');
+... WHERE column [NOT] IN ( subquery )
+```
+
+```
+SELECT * FROM region WHERE name IN ('AMERICA', 'EUROPE');
+```
+
+```
+SELECT * FROM region WHERE name IN ('NULL', 'AMERICA', 'EUROPE');
+```
+
+```
+SELECT * FROM table_name WHERE (column1, column2) IN ((NULL, 'value1'), ('value2', 'value3'));
+```
+
+## Row comparison: OR
+
+The `OR` operator is used to filter the results of a query based on more than one condition.
+It returns a record if any of the conditions separated by OR is TRUE.
+The values in the clause are used for multiple comparisons that are combined as
+a logical `OR`. The preceding query is equivalent to the following query:
+
+```
+SELECT * FROM region WHERE name = 'AMERICA' OR name = 'EUROPE';
+```
+
+## Row comparison: NOT IN
+
+The `NOT IN` comparison operator in SQL is used to exclude the rows that match any value in a list or subquery.
+
+You can negate the comparisons by adding `NOT`, and get all other regions
+except the values in list:
+
+```
+SELECT * FROM region WHERE name NOT IN ('AMERICA', 'EUROPE');
+```
+
+When using a subquery to determine the values to use in the comparison, the
+subquery must return a single column and one or more rows.
+
+```
```

Comment:
This code example might need different formatting. This screenshot shows how it looks when I build it locally. 

<img width="822" alt="Screenshot 2023-09-05 at 3 59 36 PM" src="https://github.com/prestodb/presto/assets/7013443/e593c81a-ff30-42c9-906f-565c48d56d83">